dotCMS/src/integration-test/java/com/dotcms/rendering/velocity/services/HTMLPageAssetRenderedTest.java

```diff
@@ -282,7 +320,8 @@ public static void restore() throws Exception{
      *
      */
     @Test
-    public void ContentFallbackFalse_PageFallbackTrue_PageEnglish_ViewEnglishContent1And2_ViewSpanishContent2And3() throws Exception{
+    @UseDataProvider("cases")
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The String literal "cases" appears 13 times in this file; the first occurrence is on line 323](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5025402)